kubernetes/linux/setup.sh

```diff
@@ -37,7 +37,7 @@ mv /usr/lib/ruby/gems/3.1.0/specifications/default/uri-0.11.0.gemspec /usr/lib/r
 gem uninstall time --version 0.2.0
 gem uninstall uri --version 0.11.0
 
-sudo tdnf install -y azure-mdsd-1.29.4
+sudo tdnf install -y azure-mdsd-1.29.7
```

Comment:
1. can you please double check this version works on both ARM64 and x64 nodes?
2.  Lets ask the MDSD team to have the release notes for this version so we and other team can refer the release notes for this. In the past, AKS team indicated that, we were using not released version by looking at MDSD release notes.

Reply:
1) confirmed with Aaron that it's for both
2) ack